core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -78,16 +130,16 @@ object ZkUtils extends Logging {
   }
 
   def getEntityConfigRootPath(entityType: String): String =
-    EntityConfigPath + "/" + entityType
+    ZkUtils.EntityConfigPath + "/" + entityType
```

Comment:
It seems that all methods like getEntityConfigRootPath should be in the ZkUtils object?